src/modules/getLikeData.js

```diff
@@ -0,0 +1,11 @@
+import displayLikes from './displayLikes.js';
+
+const url = 'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XStiGEvTbIVhBJcUTGCN/likes/';
+const getLikes = async () => {
+  let likes = [];
+  const response = await fetch(url);
+  likes = await response.json();
+  displayLikes(likes);
+};
+
+export default getLikes;
```

Comment:
You have done excellently well by fetching the data from the API 👏👏 but I would suggest that you include an error handler like the try/catch. This would ensure that errors do not break your application if there are any.

Reply:
Thanks for the review, I would implement these changes